v1.go

```diff
@@ -0,0 +1,16 @@
+package plugin
+
+// Plugin is the interface every plugin need to implement
+type Plugin interface {
+	// Enable is called every time a plugin is started. Spawn custom goroutines here for polling, etc. It is always called after ^Register.*Handler$
+	Enable() error
```

Comment:
Is this called for each user or only globally?

Reply:
For each user. The loader will keep track of all Plugin instances.